Dependencies/xr/Source/OpenXR/Include/XrContextHelperOpenXR.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#if __has_include("jsi/jsi.h")

```

Comment:
It didn't click for me earlier when we chatted that this would be in BabylonNative. Would it make more sense to just include this file in BabylonReactNative? It's a bit weird to split it that way, but it's also a bit weird to have any publicly exposed interface that uses jsi directly in BabylonNative.

Reply:
moving to BabylonReactNative